app/src/main/kotlin/it/vfsfitvnm/vimusic/ui/screens/AlbumScreen.kt

```diff
@@ -243,17 +243,6 @@ fun AlbumScreen(browseId: String) {
                                 .clickable {
                                     menuState.display {
                                         Menu {
-                                            MenuEntry(
```

Comment:
this is album screen. Do we need to remove the option here? "get rid of the enqueue button on the playlists"
I dont think so

Reply:
Yes you are right, I didn't notice that we have to get rid of enqueue button only on playlists